core/io/dir_access.h

```diff
@@ -116,10 +116,10 @@ class DirAccess : public RefCounted {
 		Ref<DirAccess> da = create(ACCESS_FILESYSTEM);
 		if (da->file_exists(p_path)) {
 			if (da->remove(p_path) != OK) {
-				ERR_FAIL_MSG("Cannot remove file or directory: " + p_path);
+				ERR_FAIL_MSG("Cannot remove file or directory: " + p_path + ".");
 			}
 		} else {
-			ERR_FAIL_MSG("Cannot remove non-existent file or directory: " + p_path);
+			ERR_FAIL_MSG("Cannot remove non-existent file or directory: " + p_path + ".");
```

Comment:
I think in this case, the omission was intention, to avoid confusion at to whether the dot is part of the file path or not.

Reply:
Oops, yeah that makes sense. I should avoid doing it to file paths. Agreed.

Reply:
There are existing examples of adding the period at the end of the path. Would you like me to remove those? 

https://github.com/godotengine/godot/blob/87d40ba743ff3a3a484db051c11b16559604efa9/core/config/project_settings.cpp#L1096

Reply:
What's most commonly used currently?

Reply:
Without the dot is used 3x as much. 

![image](https://github.com/godotengine/godot/assets/105675984/12275d00-4113-426d-917c-73a015656b5d)

vs

![image](https://github.com/godotengine/godot/assets/105675984/6578d8a8-c584-4d65-97b0-aa459de0d267)

Reply:
Let's standardize on no dots for these then.